.gitignore

```
@@ -35,10 +32,15 @@ GPATH
 GRTAGS
 GSYMS
 GTAGS
-examples/ex1
-examples/feti/ex1
-examples/jbearing2
+/src/**/tests/**/ex[0-9]*
+/src/**/tutorials/**/ex[0-9]*
+/src/tutorials/jbearing2
 *.h.html
 *.c.html
 makefile.html
 index.html
+/arch-*/
+/lib/
```

This excludes any additions to lib/permon/
```suggestion
/lib/*
!/lib/permon/
/lib/permon/conf/files
```

---

Oh gotcha, I forgot there some non-generated stuff as well :-) Thanks.